Libs/Mesh/Mesh.h

```diff
@@ -14,8 +15,10 @@ class Mesh
   using MeshType = vtkSmartPointer<vtkPolyData>;
 
   Mesh(const std::string& pathname) : mesh(read(pathname)) {}
-  Mesh(vtkSmartPointer<vtkPolyData>&& rhs) : mesh(std::move(rhs)) {}
-
+  // Mesh(vtkSmartPointer<vtkPolyData>&& rhs) : mesh(std::move(rhs)) {}
```

Comment:
These are significant modifications that need more consideration. For example, copying a smart pointer means the previous owner can keep modifying the object, resulting in difficult bugs, particularly when the class is used by other apps, python apis, etc. Thus we intentionally provided only a move constructor.